### PR TITLE
NO-ISSUE - Migrations unit tests database cleanup

### PR DESCRIPTION
**Pull request by omertuc**
Migration unit tests didn't delete the test DB they created,
now they will.

/cc @carbonin

### CODE REVIEW

File: internal/migrations/migrations_suite_test.go [hunk 1]
```diff
@@ -53,6 +53,8 @@ func getColumnType(db *gorm.DB, model interface{}, column string) (string, error
 	colTypes, err := rows.ColumnTypes()
 	Expect(err).NotTo(HaveOccurred())
 
+	Expect(rows.Close()).NotTo(HaveOccurred())
```
Comment (omertuc) [inline internal/migrations/migrations_suite_test.go:56]: @carbonin

Comment (omertuc) [reply internal/migrations/migrations_suite_test.go:56]: image

File: internal/migrations/migrations_suite_test.go [hunk 2]
```diff
@@ -49,10 +49,14 @@ func migrateTo(db *gorm.DB, migratoinID string) error {
 func getColumnType(db *gorm.DB, model interface{}, column string) (string, error) {
 	rows, err := db.Model(model).Rows()
 	Expect(err).NotTo(HaveOccurred())
+	defer func() {
+		Expect(rows.Close()).NotTo(HaveOccurred())
```
Comment (carbonin) [inline internal/migrations/migrations_suite_test.go:53]: :facepalm: 

Can you make this `Expect(rows.Close()).To(Succeed())`?

**Comment by carbonin** [comment]
Thanks for this find @omertuc I think you'll need a rebase now that https://github.com/openshift/assisted-service/pull/1380 is merged though.

**Review by carbonin — APPROVED** [review]

**Comment by carbonin** [comment]
/hold

To reduce spam until lint is fixed

**Pull request merged**